sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v2_7/q51a.sf100/explain.txt

```diff
@@ -1,74 +1,72 @@
-               :                 :                                +- ReusedExchange (16)
-               :                 +- * Sort (54)
-               :                    +- Exchange (53)
-               :                       +- * HashAggregate (52)
-               :                          +- * HashAggregate (51)
-               :                             +- * Project (50)
-               :                                +- * SortMergeJoin Inner (49)
-               :                                   :- * Sort (42)
-               :                                   :  +- Exchange (41)
-               :                                   :     +- * Project (40)
-               :                                   :        +- Window (39)
-               :                                   :           +- * Sort (38)
-               :                                   :              +- Exchange (37)
-               :                                   :                 +- * HashAggregate (36)
-               :                                   :                    +- Exchange (35)
-               :                                   :                       +- * HashAggregate (34)
-               :                                   :                          +- * Project (33)
-               :                                   :                             +- * BroadcastHashJoin Inner BuildRight (32)
-               :                                   :                                :- * Filter (30)
-               :                                   :                                :  +- * ColumnarToRow (29)
-               :                                   :                                :     +- Scan parquet default.store_sales (28)
-               :                                   :                                +- ReusedExchange (31)
-               :                                   +- * Sort (48)
-               :                                      +- Exchange (47)
-               :                                         +- * Project (46)
-               :                                            +- Window (45)
-               :                                               +- * Sort (44)
-               :                                                  +- ReusedExchange (43)
-               +- * Project (64)
-                  +- Window (63)
-                     +- * Sort (62)
-                        +- ReusedExchange (61)
+TakeOrderedAndProject (68)
++- * Filter (67)
+   +- * HashAggregate (66)
+      +- * HashAggregate (65)
+         +- * Project (64)
+            +- * SortMergeJoin Inner (63)
+               :- Window (58)
+               :  +- * Sort (57)
+               :     +- Exchange (56)
+               :        +- * Project (55)
+               :           +- * Filter (54)
+               :              +- SortMergeJoin FullOuter (53)
```

Comment:
`Exchange (26)` and `Exchange (53)` are no longer required.

join keys:
left: [item_sk#11, d_date#6]
right: [item_sk#37, d_date#32]

partitioning:
left: hashpartitioning(item_sk#11, 5) or hashpartitioning(item_sk#19, 5)
right: hashpartitioning(item_sk#37, 5) or hashpartitioning(item_sk#45, 5)